### DISCUSSION TITLE
unexpected failure: Platform conversion failed Erorr #3003

### DISCUSSION THREAD

**Discussion by kairusds** [airsdk/Adobe-Runtime-Support#3020] [Q&A] [answered]
Hello, for some reason whenever I try to compile my Adobe AIR app in Ubuntu 22.04, I get this error:
```
unexpected failure: Platform conversion failed. Process exited with error 5. Output was "[ErrorEvent type="error" bubbles=false cancelable=false eventPhase=2 text="Unhandled exception Error: Error #3003" errorID=3003]
"
java.io.IOException: Platform conversion failed. Process exited with error 5. Output was "[ErrorEvent type="error" bubbles=false cancelable=false eventPhase=2 text="Unhandled exception Error: Error #3003" errorID=3003]
"
        at com.adobe.air.nai.NativePackager.conversionFail(NativePackager.java)
        at com.adobe.air.nai.NativePackager.invokePlatformConversion(NativePackager.java)
        at com.adobe.air.nai.NativePackager.createPackage(NativePackager.java)
        at com.adobe.air.ADT.parseArgsAndGo(ADT.java)
        at com.adobe.air.ADT.run(ADT.java)
        at com.adobe.air.ADT.main(ADT.java)
```

Is there any way to solve this?

**Comment by ajwfrost** [comment, marked as answer]
Hi

Which type of packaging are you trying to use there? The `-target bundle` should work, but `-target native` isn't supported on Linux as we don't have a shared runtime installer for it...

If you are using the bundle option though - worth checking that the AIR SDK license is set up properly? If you run the app using ADL, does it work okay?

thanks

**Reply by kairusds** [reply]
Yes, thank you, it works great now. I didn't know that there's a `bundle` package option as I haven't seen it in a documentation.
